input/resources/au-medicarecardnumber.xml

```diff
@@ -44,6 +44,14 @@
         <valueString value="32788511952" />
       </example>
       <maxLength value="11" />
+      <condition value="inv-mcn-01"/>
+        <constraint>
+        <key value="inv-mcn-01"/>
+        <severity value="error"/>
+        <human value="Medicare Card Number shall be at least 10 digits"/>
```

Comment:
The text specifies digits, but the expression does not enforce that the content must be numeric. 
Please confirm if the numeric validation is required and either update the text or the expression to match.

Reply:
Thanks Jaymee, confirmed with Brett to check for the length as per the resolution.